site/src/components/Dashboard/DashboardProvider.tsx

```diff
@@ -112,11 +112,10 @@ export const useDashboard = (): DashboardProviderValue => {
 };
 
 export const useIsWorkspaceActionsEnabled = (): boolean => {
-  const { entitlements, experiments } = useDashboard();
+  const { entitlements } = useDashboard();
```

Comment:
I don't think we need a separate hook for this any longer as we are just checking the advanced scheduling requirement. I would take it out.